### ISSUE TITLE
Service won't store objects

### ISSUE THREAD

**Issue by cabstool** [grevory/angular-local-storage#301]
I get a JSON stringify error when attempting to store any object: error converting circular reference. I'm 100% sure my object(s) do not have circular references.

**Comment by grevory** [comment]
Is there any way you can provide more detail for reference? Is anyone else experiencing this issue?

**Comment by cabstool** [comment]
I'm using Breeze and Angular to get an entity object from my backend. The object has no references to itself or any other object at all. When I call localStorageService.set("name", object) I get the JSON.stringify error saying that it cannot convert an object containing a circular reference...

**Comment by cabstool** [comment]
here is my client side object i'm attempting to store:

{
    "$id": "1",
    "$type": "mydomain.Models.Location, mydomain",
    "Zip": "99999",
    "City": "city",
    "State": "state"
  },

it's a Breeze entity object which came over from my Breeze server.....

**Comment by cabstool** [comment]
Hi Gregory,

Any progress on this bug? Thanks.

**Comment by grevory** [comment]
No, I am not working on bugs at the moment. Feel free to submit a PR.

**Comment by akanshgulati** [comment]
The current work around for this is to stringify the object. Once you want to get the object, there is no need of parsing the content. It automatically, gives you object.
